SmartDeviceLink/public/SDLAudioData.h

```diff
@@ -0,0 +1,55 @@
+//
+//  SDLAudioData.h
+//  SmartDeviceLink
+//
+//  Created by Nicole on 11/9/20.
+//  Copyright © 2020 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "SDLSpeechCapabilities.h"
+
+@class SDLFile;
+@class SDLTTSChunk;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLAudioData : NSObject <NSCopying>
+
+/// The text-to-speech prompts that will used and/or audio files that will be played. The audio prompts and files will be played in the same order they are added.
+@property (nullable, copy, nonatomic, readonly) NSArray<SDLTTSChunk *> *audioData;
```

Comment:
To make this simpler, you could make a mutable version of this array in the .m interface extension. Then make a getter for this property that just returns `[mutableArray copy]`. Then you can make some methods where you're creating a temp mutable array simpler.